srcpkgs/nrpe/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'nrpe'
+pkgname=nrpe
+version=3.2.1
+revision=1
+build_style=gnu-configure
+configure_args="--bindir=/usr/bin --with-pkgsysconfdir=/etc/nagios --with-pluginsdir=/usr/lib/monitoring-plugins --with-nagios-user=_nagios --with-nagios-group=_nagios --with-nrpe-user=_nagios --with-nrpe-group=_nagios"
```

Comment:
Please cut at 80 columns and put in a newline starting a whitespace

Reply:
Also --bindir is redundant as is already present in the gnu-configure build-style